sigle/sigle
# feat: add next themes

Adds dark mode with feature flag enabled.

Steps to test @QuentinSaubadu :

- Add `?experimentalThemeToggle=true` to the end of the dashboard url
- On any page containing the app header, the Theme Toggle button should be visible.
- Pages to test are the login and register username page, editor page, and public story page. 

Closes #374 #375  #376 #377
